docs/guides/universal-profile/check-if-address-is-universal-profile.md

```diff
@@ -12,10 +16,41 @@ You may want to read the [Standard Detection](../../standards/standard-detection
 
 To check if an address is a [Universal Profile](../../standards/universal-profile/introduction.md) we need to perform 3 checks:
 
-## 1) Check address format
+## Setup
+
+Make sure you have the following dependencies installed before beginning this tutorial.
+
+- You can use either [`web3.js`](https://github.com/web3/web3.js) or [`ethers.js`](https://github.com/ethers-io/ethers.js/)
+- You MUST install [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts/)
```

Comment:
we have this exact setup part on every single page. Might be better to add a "pre requisite" page or global "setup page" at the top of the "guides" section and simply make a reference to it imo